packages/vscode-extension/package.nls.json

```diff
@@ -256,6 +258,13 @@
     "teamstoolkit.localDebug.failedCheckers": "Unable to check: [%s].",
     "teamstoolkit.handlers.askInstallOfficeAddinDependency": "Install dependencies for Office Add-in?",
     "teamstoolkit.handlers.installOfficeAddinDependencyCancelled": "Dependency installation is canceled, but you can install dependencies manually by clicking the 'Development - Check and Install Dependencies' button on the left side.",
+    "teamstoolkit.handlers.askInstallCopilot": "To use Github Copilot, install its extension first.",
+    "teamstoolkit.handlers.askInstallCopilot.install": "Install",
+    "teamstoolkit.handlers.askInstallCopilot.cancel": "Cancel",
+    "teamstoolkit.handlers.installExtension.output": "You need to install %s following \"%s\" first.",
+    "teamstoolkit.handlers.installCopilotError": "Unable to install Github Copilot Chat. Install it following %s and try again.",
+    "teamstoolkit.handlers.chatTeamsAgentError": "Unable to automatically focus Github Copilot Chat. Open Github Copilot Chat and start with \"%s\"",
+    "teamstoolkit.handlers.verifyCopilotExtensionError": "Unable to verify Github Copilot Chat. Install it manually following %s and try again.",
     "teamstoolkit.localDebug.learnMore": "Learn More",
```

Comment:
Proposed: Get more info
Logic: As per our style guide, we don't use 'Learn more'.